02-electric/electric-section-one/electric-section-one.css

```diff
@@ -1,14 +1,28 @@
 .electric-section-one-container {
   padding: var(--container-padding-mobile);
+  display: grid;
+}
```

Comment:
define the grid-template-columns to 1fr here
because if we changed layout later we would need it defined in the mobile first
so it would revert back and forward between them on media query.

Reply:
done